bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/HBaseRequestAdapter.java

```diff
@@ -271,4 +332,12 @@ protected String getTableNameString() {
     return getBigtableTableName().toString();
   }
 
+  private MutateRowsRequest.Entry toEntry(RowMutation rowMutation) {
+    return rowMutation.toBulkProto(requestContext).getEntries(0);
```

Comment:
This seems to be used in one place, not sure if its worth having a method for this

Reply:
Its used for Put, Delete and RowMutations. Also this will go away once we adapt the return type to model.